model/registry/k8s.yaml

```diff
@@ -0,0 +1,210 @@
+groups:
+  - id: registry.k8s.cluster
```

Comment:
Do we need to have the subgroups in the registry? How about having one plain list here, for simplicity?